tests/unit/client_test.py

```diff
@@ -1,5 +1,6 @@
 import datetime
 import docker
+from docker.utils import kwargs_from_env
```

Comment:
I know I could have just used `docker.from_env()` in the test, but I wanted something as close as possible to what users of the old docker-py would actually have written. Let me know if you think this is silly.